NAMESPACE

```diff
@@ -11,5 +11,6 @@ export(credentials_gce)
 export(credentials_service_account)
 export(credentials_travis)
 export(credentials_user_oauth2)
+export(tidyverse_app)
```

Comment:
Given that we don't intend for *most* users to use this directly, should we instead leave this unexported and let tidyverse packages do a `:::` to find it? (That seems really dirty, but am curious if there's an accepted R idiom for this sort of thing.)